nimare/meta/kernel.py

```diff
@@ -243,21 +262,30 @@ def _transform(self, mask, coordinates):
             if self.fwhm is not None:
                 assert np.isfinite(self.fwhm), "FWHM must be finite number"
                 if self.fwhm not in kernels.keys():
-                    _, kern = get_ale_kernel(mask, fwhm=self.fwhm)
+                    _, kern = get_ale_kernel(
+                        mask,
+                        fwhm=self.fwhm,
+                    )
                     kernels[self.fwhm] = kern
                 else:
                     kern = kernels[self.fwhm]
             else:
                 assert np.isfinite(sample_size), "Sample size must be finite number"
                 if sample_size not in kernels.keys():
-                    _, kern = get_ale_kernel(mask, sample_size=sample_size)
+                    _, kern = get_ale_kernel(
+                        mask,
+                        sample_size=sample_size,
+                    )
                     kernels[sample_size] = kern
                 else:
                     kern = kernels[sample_size]
             kernel_data = compute_ale_ma(mask.shape, ijk, kern)
-            transformed.append((kernel_data, id_))
+            transformed[i_exp, :, :, :] = kernel_data
```

Comment:
It also seems to slow things down substantially, which I guess is to be expected, but is a serious problem for standard meta-analyses (esp. ALEs) that _were_ fine without low-memory tools before, but are running out of memory without those tools now.

Reply:
I might be misunderstanding what you're saying, and haven't looked at the code, but if optimal behavior is dependent on the setting of `low_memory`, can we just add a check for that, and use the previous approach for `low_memory=False` and the new one for `low_memory=True`?

Reply:
I've updated it so that `low_memory=False` is the same as before, so the memory burden should be the same as before. The code doesn't look great, but it should all work now.